### PR TITLE
fix(canary/gcs): Explicitly set secret session manager.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
Fixes https://github.com/spinnaker/spinnaker/issues/4312.

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java [hunk 1]
```diff
@@ -30,16 +30,13 @@
...
 import org.springframework.scheduling.TaskScheduler;
-import org.springframework.stereotype.Component;
 
 @Data
 @EqualsAndHashCode(callSuper = false)
-@Component
 public class GoogleCanaryAccountValidator extends CanaryAccountValidator {
 
-  @Autowired
+  @Setter
```
Comment (jtk54) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java]: Do you need an `@Setter` if the class has `@Data`?

Comment (duftler) [reply halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java]: Nope, fixed.

**Comment by duftler** [comment]
@spinnakerbot cherry-pick 1.19

**Comment by duftler** [comment]
@spinnakerbot cherry-pick 1.18

**Pull request merged**